osquery/config/parsers/options.cpp

```diff
@@ -80,6 +80,11 @@ Status OptionsConfigParserPlugin::update(const std::string& source,
       value = std::to_string(option.value.GetInt());
     } else if (option.value.IsNumber()) {
       value = std::to_string(option.value.GetUint64());
+    } else if (option.value.IsObject() || option.value.IsArray()){
```

Comment:
It would be great to make a test for it. To be sure it works and none will break it in the future by occasional changes.

Reply:
done